data/classificator.txt

```diff
@@ -318,6 +329,7 @@ world  +
             permissive  -
             tunnel  -
             {}
+        traffic_signals  -
```

Comment:
Светофоры мы когда-то рисовали, убрали из-за замусоривания. Но добавить только на последних зумах может быть очень полезно для роутинга, чтобы правильно предсказывать время.
А мусорки на последнем зуме нужны по-любому, что тут обсуждать?

> On Apr 14, 2016, at 18:08, Viktor Govako <notifications@github.com> wrote:
> 
> In data/classificator.txt <https://github.com/mapsme/omim/pull/2884#discussion_r59734524>:
> 
> > @@ -318,6 +329,7 @@ world  +
> >              permissive  -
> >              tunnel  -
> >              {}
> > +        traffic_signals  -
> Опять таки, чтобы их нормально добавлять, их надо начать рисовать ...
> Лично я крайне сомневаюсь в целесообразности этого.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2884/files/29ca9e7b6cf0dda705a66e9d24ff265283f6dcac#r59734524>